### ISSUE TITLE
Getting Started > Designers > #a11y contrast issue

### ISSUE THREAD

**Issue by arbydev** [carbon-design-system/carbon#338]
http://carbondesignsystem.com/getting-started/designers
The first paragraph that says "The Carbon Design Kit is a living..."  the blue text does not meet AA requirements for contrast. This blue color is used throughout the site.

**Comment by bsonefeld** [comment]
Hi @arbydev , we've updated our color to meet WCAG AA color contrast guidelines. You can see the change reflected here: http://carbondesignsystem.com/getting-started/designers
